ext/zlib/php_zlib.h

```diff
@@ -47,6 +47,7 @@ typedef struct _php_zlib_context {
 	int status;
 	size_t inflateDictlen;
 	php_zlib_buffer buffer;
+	zend_object std;
 } php_zlib_context;
```

Comment:
Apparently, the same data structure is used during inflation and deflation... Which leads to the question if I should separate it to a `zlib_inflate_context` and a `zlib_deflate_context`?